docs/data/gridftp.md

```diff
@@ -174,18 +183,72 @@ In addition to the GridFTP service itself, there are a number of supporting serv
 | GridFTP   | `globus-gridftp-server`               |                                                                                        |
 
 
-Validating GridFTP
+Start the services in the order listed and stop them in reverse order. As a reminder, here are common service commands (all run as `root`):
+
+| To...                                   | On EL6, run the command...                  | On EL7, run the command...                      |
+| :-------------------------------------- | :----------------------------------------   | :--------------------------------------------   |
+| Start a service                         | `service <SERVICE-NAME> start` | `systemctl start <SERVICE-NAME>`   |
+| Stop a  service                         | `service <SERVICE-NAME> stop`  | `systemctl stop <SERVICE-NAME>`    |
+| Enable a service to start on boot       | `chkconfig <SERVICE-NAME> on`  | `systemctl enable <SERVICE-NAME>`  |
+| Disable a service from starting on boot | `chkconfig <SERVICE-NAME> off` | `systemctl disable <SERVICE-NAME>` |
+
+
+Validation
 ------------------
 
-The GridFTP service can be validated by using globus-url-copy. You will need to run `grid-proxy-init` or `voms-proxy-init` in order to get a valid user proxy in order to communicate with the GridFTP server.
+### GridFtp
+
+
+1. Acquire a user certificate [opensciencegrid.org/docs/security/user-certs](http://opensciencegrid.org/docs/security/user-certs/)
+1. Find your subject DN
+
+        :::console
+        user@host # openssl x509 -in <CERITIFICATE_FILE.pem> -noout -subject
+
+1. Add your DN to `/etc/grid-security/grid-mapfile` and map it to a non-root user, see:
+[/opensciencegrid.org/docs/security/lcmaps-voms-authentication/#mapping-users]( https://opensciencegrid.org/docs/security/lcmaps-voms-authentication/#mapping-users)
+1. As the non-root user, generate your proxy
+
+        :::console
+        user@host # voms-proxy-init
+
+1. Create a test file to be transfered
+
+        :::console
+        user@host # echo "Hello World!" > /tmp/hello_world
+
+1. Transfer the file we just created
+
+        :::console
+        user@host # globus-url-copy file:///tmp/hello_world gsiftp://yourhost.yourdomain/tmp/hello_world
+
+1. To verify that the authentication is working, we could remove our proxy and execute the last command again, this time it should fail.
+
+        :::console
+        user@host # voms-proxy-destroy
+        user@host # globus-url-copy file:///tmp/hello_world gsiftp://yourhost.yourdomain/tmp/hello_world
+
+!!!note
+     Keep in mind that when invoked as root, `globus-url-copy` will attempt to use the host certificate instead of your user certificate, which could produce confusing results.
```

Comment:
```suggestion
     Keep in mind that when invoked as root, `globus-url-copy` will attempt to use the host certificate instead of your user certificate, which may produce confusing results.
```